backend/apis/clients/client-interactions/create-client-interaction.api.js

```diff
@@ -0,0 +1,99 @@
+const {
+  app,
+  databaseError,
+  pool,
+  invalidRequest,
+  notFound
+} = require("../../../server");
+
+const mysql = require("mysql");
+
+const {
+  checkValid,
+  validId,
+  validEnum,
+  nonEmptyString,
+  validDate,
+  validTime
+} = require("../../utils/validation-utils");
+
+const { createResponseInteractionObject } = require("./interaction-log.utils");
+
+const { createResponseLogObject } = require("./activity-log.utils");
+
+app.post("/clients/:clientId/interactions", (req, res) => {
+  const validationErrors = [
+    ...checkValid(req.params, validId("clientId")),
+    ...checkValid(req.params, validId("serviceId")),
```

Comment:
`serviceId` is not in the params. The endpoint is `"/clients/:clientId/interactions"` which does not have a `serviceId` in it.